### ISSUE TITLE
[Bug...?] Flywheel 1.18-0.6.5 is considered too low of a version despite being above 1.18-0.6.2

### ISSUE THREAD

**Issue by Corvus20** [TeamMoegMC/SteamPowered#43]
### Error Behaviour

Steam Powered does not register flywheel's version correctly and thinks that it is invalid. This is actually a problem across multiple other create addon mods as of lately (Big Cannons and Create Crafts & Additions), should be a relatively simple fix, I would hope.

Screenshot of error on loading error screen:
![image](https://user-images.githubusercontent.com/75586106/184685771-5974fcc0-22fd-4254-b048-0124a17f9b53.png)

### Expected Behaviour

Steam Powered should register flywheel 1.18-0.6.5 correctly as meeting the requirements and load normally.

### Reproduce

1. Install Flywheel 1.18-0.6.5, Create 1.18.2-0.5.0.d, and Steam Powered 1.18.2-2.0.3-alpha.
2. Launch game.
3. Wait for game to progress to the "errors occurred while loading" screen.
4. See error.

### Crash Report

https://pastebin.com/YTg1bs70

### Mod version

1.18.2-2.0.3-alph

### Server version

N/A

### Other mods and plugins

Initially experienced on a much larger custom modpack, but was also tested in a standalone instance with just the necessary mods and got the same result.

**Comment by TwiningShiningDawn** [comment]
also having this issue, any assistance would help

**Comment by dreaderinyes** [comment]
Also having this issues, which is a huge disappointment.

**Comment by isaaclepes** [comment]
It looks like the version numbering of flywheel changed.  Internally it calls itself '0.6.5' where SteamPowered is looking for '1.18-0.6.5' for example. I will modify the mod.toml to test this and report back.

**Comment by isaaclepes** [comment]
Update: changing the toml did allow it to attempt to load, however it is failing on Create 0.5d due to them removing Registrate

`[16:20:45] [main/WARN]: Error loading class: com/simibubi/create/content/contraptions/components/flywheel/FlywheelTileEntity (java.lang.ClassNotFoundException: com.simibubi.create.content.contraptions.components.flywheel.FlywheelTileEntity)
[16:20:45] [main/WARN]: @Mixin target com.simibubi.create.content.contraptions.components.flywheel.FlywheelTileEntity was not found steampowered.mixins.json:FlywheelTileEntityAccess
[16:20:45] [main/WARN]: Error loading class: com/simibubi/create/content/contraptions/components/flywheel/FlywheelTileEntity (java.lang.ClassNotFoundException: com.simibubi.create.content.contraptions.components.flywheel.FlywheelTileEntity)
[16:20:45] [main/WARN]: @Mixin target com.simibubi.create.content.contraptions.components.flywheel.FlywheelTileEntity was not found steampowered.mixins.json:MixinFlywheel
[16:20:46] [main/WARN]: Error loading class: snownee/snow/WorldTickHandler (java.lang.ClassNotFoundException: snownee.snow.WorldTickHandler)
[16:20:46] [main/WARN]: Error loading class: snownee/snow/WorldTickHandler (java.lang.ClassNotFoundException: snownee.snow.WorldTickHandler)
[16:20:46] [main/WARN]: Error loading class: snownee/snow/WorldTickHandler (java.lang.ClassNotFoundException: snownee.snow.WorldTickHandler)
[16:20:48] [main/WARN]: @Inject(@At("HEAD")) Shift.BY=1 on blueprint.mixins.json:EntityMixin::handler$bga000$baseTick exceeds the maximum allowed value: 0. Increase the value of maxShiftBy to suppress this warning.
[16:20:48] [pool-3-thread-1/WARN]: Error loading class: snownee/snow/WorldTickHandler (java.lang.ClassNotFoundException: snownee.snow.WorldTickHandler)
[16:20:50] [pool-3-thread-1/WARN]: Error loading class: com/tterrag/registrate/util/entry/BlockEntry (java.lang.ClassNotFoundException: com.tterrag.registrate.util.entry.BlockEntry)
`

**Comment by rmichaluszek** [comment]
Same issue, any ideas?

**Comment by DUDEbehindDUDE** [comment]
> Update: changing the toml did allow it to attempt to load, however it is failing due to them removing Registrate from flywheel/create.
> 
> `[16:20:45] [main/WARN]: Error loading class: com/simibubi/create/content/contraptions/components/flywheel/FlywheelTileEntity (java.lang.ClassNotFoundException: com.simibubi.create.content.contraptions.components.flywheel.FlywheelTileEntity) [16:20:45] [main/WARN]: @Mixin target com.simibubi.create.content.contraptions.components.flywheel.FlywheelTileEntity was not found steampowered.mixins.json:FlywheelTileEntityAccess [16:20:45] [main/WARN]: Error loading class: com/simibubi/create/content/contraptions/components/flywheel/FlywheelTileEntity (java.lang.ClassNotFoundException: com.simibubi.create.content.contraptions.components.flywheel.FlywheelTileEntity) [16:20:45] [main/WARN]: @Mixin target com.simibubi.create.content.contraptions.components.flywheel.FlywheelTileEntity was not found steampowered.mixins.json:MixinFlywheel [16:20:46] [main/WARN]: Error loading class: snownee/snow/WorldTickHandler (java.lang.ClassNotFoundException: snownee.snow.WorldTickHandler) [16:20:46] [main/WARN]: Error loading class: snownee/snow/WorldTickHandler (java.lang.ClassNotFoundException: snownee.snow.WorldTickHandler) [16:20:46] [main/WARN]: Error loading class: snownee/snow/WorldTickHandler (java.lang.ClassNotFoundException: snownee.snow.WorldTickHandler) [16:20:48] [main/WARN]: @Inject(@At("HEAD")) Shift.BY=1 on blueprint.mixins.json:EntityMixin::handler$bga000$baseTick exceeds the maximum allowed value: 0. Increase the value of maxShiftBy to suppress this warning. [16:20:48] [pool-3-thread-1/WARN]: Error loading class: snownee/snow/WorldTickHandler (java.lang.ClassNotFoundException: snownee.snow.WorldTickHandler) [16:20:50] [pool-3-thread-1/WARN]: Error loading class: com/tterrag/registrate/util/entry/BlockEntry (java.lang.ClassNotFoundException: com.tterrag.registrate.util.entry.BlockEntry) `

they have to update the mod to support the new cerate version first

**Comment by isaaclepes** [comment]
> > Update: changing the toml did allow it to attempt to load, however it is failing due to them removing Registrate from flywheel/create.

> they have to update the mod to support the new create version first

Yeah, I was just proving the point that the version number itself wasn't the solution.  A lot of mods broke when they removed Registrate from Create/Flywheel

https://github.com/tterrag1098/Registrate

**Comment by SortaGoodEl1** [comment]
I am also having this same issue this is frustrating

**Comment by Vatruvius** [comment]
I too am experiencing this particular issue, luckily Steam Powered is the only Create addon I have affected by the removal of Registrate from Flywheel. As I am trying to put a modpack together, and fix the startup crash reports, I am going to temporarily disable Steam Powered until I hear any progress being made from this thread or see an update on CurseForge.

**Comment by MagmaBro123** [comment]
Same issue.

**Comment by MagmaBro123** [comment]
Oof and ugh.

**Comment by MagmaBro123** [comment]
> FYI - I saw on their discord that team is focusing on The Winter Rescue at the moment, so Steam Powered updates will be a while.

I mean, updating a dependency shouldn't be that hard tbh...

**Comment by MagmaBro123** [comment]
Oh. Well, I'm not a programmer, so, sorry.

**Comment by Normal69** [comment]
Okay, then I don't need to fuss around with this.
Needs complicated solution, takes time.
Thank you.

**Comment by IsaacAMurray** [comment]
I'm also having this issue and am wondering if there has been anything new since this issue page was created as it is now March of 2023 and this month is close to an end. Surely something has to have changed...
